src/Microsoft.AspNetCore.SystemWebAdapters/Build/Microsoft.AspNetCore.SystemWebAdapters.props

```diff
@@ -0,0 +1,3 @@
+<Project>
+  <PropertyGroup />
```

Comment:
I don't think you need an empty property group, just the open and closing tags of project are good enough